templates/web-deployment.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           imagePullPolicy: {{ .Values.web.image.pullPolicy }}
           env:
             - name: TEMPORAL_GRPC_ENDPOINT
-              value: "{{ include "temporal.fullname" . }}-frontend.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.server.frontend.service.port }}"
```

Comment:
might be better just removing the `cluster.local` as that's the configurable bit

Reply:
Fixed.